iac/stacks/api.py

```diff
@@ -298,6 +301,17 @@ def __init__(
         # allow API to read/write to the public bucket
         public_bucket.grant_read_write(service.task_definition.task_role)
 
-        # Allow Image Worker to write to image bucket
-        image_processing_bucket.grant_write(image_worker.task_definition.task_role)
-        image_processing_bucket.grant_read_write(service.task_definition.task_role)
+        if config.env_id == "dev":
+            # Create classification image bucket
+            image_processing_bucket = s3.Bucket(
+                self,
+                id="MermaidImageProcessingBackupBucket",
+                bucket_name=image_bucket_name,
+                removal_policy=RemovalPolicy.RETAIN,
+                public_read_access=False,
```

Comment:
I was thinking this would need to be public so frontend can hit image urls directly; are you thinking differently?

Reply:
They should be provided a signed url by django storages

Reply:
Hmm but if it's a Public Data bucket that any ML researcher can use without association with MERMAID - ?